lib/pg_ltree/ltree.rb

```diff
@@ -96,7 +97,11 @@ def ltree_path
       #
       # @return [String] ltree previous value
       def ltree_path_was
-        public_send :"#{ltree_path_column}_was"
+        if active_record_5_1_or_above?
```

Comment:
This code checks rails version for each call, but this should be done once time on the app starting.